netbox/templates/core/plugin.html

```diff
@@ -0,0 +1,154 @@
+{% endblock breadcrumbs %}
+
+{% block subtitle %}
+  <div class="text-secondary fs-5">
+    {{ plugin.tag_line }}
+    <a href="{{ plugin.homepage_url }}" target="_blank">Learn more <i class="mdi mdi-launch"></i></a>
+  </div>
+{% endblock subtitle %}
+
+{% block controls %}
+{% endblock %}
+
+{% block tabs %}
+  <ul class="nav nav-tabs" role="tablist">
+    <li class="nav-item" role="presentation">
+      <a class="nav-link active" id="overview-tab" data-bs-toggle="tab" data-bs-target="#overview" type="button" role="tab" aria-controls="edit-form" aria-selected="true">
+        {% trans "Overview" %}
+      </a>
+    </li>
+    {% if not plugin.is_local %}
+      <li class="nav-item" role="presentation">
+        <button class="nav-link" id="version-history-tab" data-bs-toggle="tab" data-bs-target="#version-history" type="button" role="tab" aria-controls="object-list" aria-selected="false">
+          {% trans "Version history" %}
+        </button>
+      </li>
+      {% if 'commercial' not in settings.RELEASE.features %}
+        <li class="nav-item" role="presentation">
+          <button class="nav-link" id="install-tab" data-bs-toggle="tab" data-bs-target="#install" type="button" role="tab" aria-controls="object-list" aria-selected="false">
+            {% trans "Install" %}
+          </button>
+        </li>
+      {% endif %}
+    {% endif %}
+  </ul>
+
+{% endblock tabs %}
+
+{% block content %}
+
+  <div class="tab-pane show active" id="overview" role="tabpanel" aria-labelledby="overview-tab">
+
+  <div class="row">
+    <div class="col col-md-12">
+
+      <div class="card">
+        <h5 class="card-header">{% trans "Author" %}</h5>
+        <table class="table table-hover attr-table">
+          <tr>
+            <th scope="row">{% trans "Author" %}</th>
+            <td>{{ plugin.author.name }}</td>
+          </tr>
+          <tr>
+            <th scope="row">{% trans "Org id" %}</th>
+            <td>{{ plugin.author.org_id|placeholder }}</td>
+          </tr>
+          <tr>
+            <th scope="row">{% trans "URL" %}</th>
+            <td>{{ plugin.author.url|urlize|placeholder }}</td>
+          </tr>
+        </table>
+      </div>
+
+      <div class="card">
+        <h5 class="card-header">{% trans "Plugin" %}</h5>
+        <table class="table table-hover attr-table">
+          <tr>
+            <th scope="row">{% trans "Name" %}</th>
+            <td>{{ plugin.title_short }}</td>
+          </tr>
+          <tr>
+            <th scope="row">{% trans "Tag line" %}</th>
+            <td>{{ plugin.tag_line|placeholder }}</td>
+          </tr>
+          <tr>
+            <th scope="row">{% trans "License" %}</th>
+            <td>{{ plugin.license_type }}</td>
+          </tr>
+          <tr>
+            <th scope="row">{% trans "Description" %}</th>
+            <td>{{ plugin.description_short|markdown }}</td>
+          </tr>
+          <tr>
+            <th scope="row">{% trans "URL" %}</th>
```

Comment:
This row is redundant with the _Learn more_ link at the top of the page. Please remove.